packages/cli/README.md

```diff
@@ -3615,7 +3615,7 @@ EXAMPLES
   $ smartthings edge:channels --subscriber-type HUB --subscriber-id <hub-id>
 ```
 
-_See code: [@smartthings/plugin-cli-edge](https://github.com/SmartThingsCommunity/smartthings-cli/blob/v2.0.0-beta.1/src/commands/edge/channels.ts)_
+_See code: [@smartthings/plugin-cli-edge](https://github.com/SmartThingsCommunity/smartthings-cli/blob/@smartthings/cli@2.0.0-beta.1/packages/edge/src/commands/edge/channels.ts)_
```

Comment:
This still 404s because `@smartthings/cli@2.0.0-beta.1` doesn't exist. We might have to either copy what heroku is doing, or settle for `/tree/main/packages`... instead of pointing to the most recently released code.

Reply:
Won't that get created when the edge plugin is released? (It looks like that happens with the other packages.) Maybe I should include a changeset for this PR?

Reply:
The correct release tag is [@smartthings/plugin-cli-edge@2.0.0-beta.1](https://github.com/SmartThingsCommunity/smartthings-cli/releases/tag/%40smartthings%2Fplugin-cli-edge%402.0.0-beta.1) and thus the correct form for the above URL should be https://github.com/SmartThingsCommunity/smartthings-cli/blob/@smartthings/plugin-cli-edge@2.0.0-beta.1/packages/edge/src/commands/edge/channels.ts

Reply:
Oh, I see now. Fixed.

I also added a changeset so it will make a release.